NewDawn/NewDawn/ViewControllers/SettingPageViewControllers/EditProfileViewController.swift

```diff
@@ -67,12 +67,47 @@ class EditProfileTabelViewController: UITableViewController{
     @objc func back(sender: UIBarButtonItem) {
         // Get information from local and send to server
         // Go back to the root ViewController
-        self.dismiss(animated: true, completion: {})
-        self.navigationController?.popViewController(animated: true)
+        let request = EditProfileUtil.createRegistrationRequest()
+        
+        if request == nil {
+            return
+        }
+        let activityIndicator = self.prepareActivityIndicator()
+        DispatchQueue.main.async {
```

Comment:
Should we put this inside of processSessionRequest? Otherwise it won’t do anything?

Reply:
You are right. I have deleted the line.